Dockerfile

```diff
@@ -0,0 +1,126 @@
+# Base image containing dependencies used in builder and final image
+FROM ghcr.io/swissgrc/azure-pipelines-openjdk:17.0.9.0 AS base
+
+
+# Builder image
+FROM base AS build
+
+# Make sure to fail due to an error at any stage in shell pipes
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# renovate: datasource=repology depName=debian_12/curl versioning=loose
+ENV CURL_VERSION=7.88.1-10+deb12u4
+
+RUN apt-get update -y && \
+  # Install necessary dependencies
+  apt-get install -y --no-install-recommends curl=${CURL_VERSION} && \
+  # Add Git LFS PPA
+  curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash && \
+  # Add NodeJS PPA
+  curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg && \
+  NODE_MAJOR=18 && \
+  echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_$NODE_MAJOR.x nodistro main" | tee /etc/apt/sources.list.d/nodesource.list && \
+  # Add .NET PPA
+  curl -o /tmp/packages-microsoft-prod.deb https://packages.microsoft.com/config/debian/12/packages-microsoft-prod.deb && \
+  dpkg -i /tmp/packages-microsoft-prod.deb && \
+  rm -rf /tmp/*
+
+
+# Final image
+FROM base AS final
+
+LABEL org.opencontainers.image.vendor="Swiss GRC AG"
+LABEL org.opencontainers.image.authors="Swiss GRC AG <opensource@swissgrc.com>"
+LABEL org.opencontainers.image.title="azure-pipelines-sonarscannermsbuild"
+LABEL org.opencontainers.image.documentation="https://github.com/swissgrc/docker-azure-pipelines-sonarscannermsbuild"
+
+# Make sure to fail due to an error at any stage in shell pipes
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+WORKDIR /
+# Copy Git LFS & NodeJS PPA keyring
+COPY --from=build /etc/apt/keyrings/ /etc/apt/keyrings
+# Copy .NET keyring
+COPY --from=build /usr/share/keyrings/ /usr/share/keyrings 
+COPY --from=build /etc/apt/trusted.gpg.d/ /etc/apt/trusted.gpg.d
+# Copy sources
+COPY --from=build /etc/apt/sources.list.d/ /etc/apt/sources.list.d
+
+# Install Git
+
+# renovate: datasource=repology depName=debian_12/git versioning=loose
```

Comment:
What is the reason for this being different than our GIT docker image?

Reply:
```
# renovate: datasource=github-tags depName=git/git extractVersion=^v(?<version>.*)$
ENV GIT_VERSION=2.43.0
```

Reply:
@Xzelsius In this image apt package of git is installed, while on the Git base image Git is build from source, since apt package is not updated often.